### PR TITLE
DIV-3742 Added two new mapping fields for amend petition flow

### PR DESCRIPTION
**Pull request by alexwakeman**
# Description

Two new fields for CCD to divorce mapping 

Fixes # DIV-3742

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java [hunk 1]
```diff
@@ -512,5 +512,11 @@
 
     @JsonProperty("D8LivedTogetherMoreTimeThanPermitted")
     private String livedTogetherMoreTimeThanPermitted;
+
+    @JsonProperty("previousCaseId")
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java]: PreviousCaseId PascalCase

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java [hunk 2]
```diff
@@ -512,5 +512,11 @@
...
+    private String previousCaseId;
+
+    @JsonProperty("previousReasonsForDivorce")
```
Comment (qzhou-hmcts) [inline src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java]: PascalCase

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java (hunk 1 above)
Comment (alexwakeman) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java]: Not against it but if you look at other fields like: https://github.com/hmcts/div-case-data-formatter/blob/dff38e5af3f5db31597f678a6981926433881dde/src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java#L498

they are camel

Comment (qzhou-hmcts) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java]: I think the only ones camel case are 'dueDate' and 'solSignStatementofTruth' the rest are PascalCase

File: src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java (hunk 2 above)
Comment (qzhou-hmcts) [reply src/main/java/uk/gov/hmcts/reform/divorce/caseformatterservice/domain/model/ccd/CoreCaseData.java]: Yup that is true...

**Pull request merged**